config/config.go

```diff
@@ -72,9 +72,9 @@ func get() (*Config, error) {
 		DefaultOffset:                           0,
 		DefaultPage:                             1,
 		DefaultSort:                             "relevance",
-		EnableCensusTopicFilterOption:           false,
-		EnableCensusPopulationTypesFilterOption: false,
-		EnableCensusDimensionsFilterOption:      false,
+		EnableCensusTopicFilterOption:           true,
+		EnableCensusPopulationTypesFilterOption: true,
+		EnableCensusDimensionsFilterOption:      true,
```

Comment:
Don't think these should be set to true by default yet.

Reply:
I was about to make the same comment, these should be false and then environment secrets containing these flags should be updated